en/_posts/2016-05-03-how-to-use-deskaas-lab.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "How to subscribe to and use the DeskaaS lab"
+categories: Labs
+author: lrivallain
+lang: en
+---
+
+## Ordering process
+
+Prerequisites: Having a manager access on ovh.com website.
```

Comment:
Beforehand, make sure to have an OVH.com account. If you don't, you may create one from https://www.ovh.com/manager/web/login/ and select "Create Account"